addons/account/models/account_journal_dashboard.py

```diff
@@ -79,13 +79,14 @@ def _query_has_sequence_holes(self):
             SELECT move.journal_id,
                    move.sequence_prefix
               FROM account_move move
-             WHERE move.journal_id = ANY(%(journal_ids)s)
```

Comment:
Why would you remove this clause?

Reply:
The spec was to only have the warning for "sale" journals

Reply:
That's not the question. You should still filter on `self.ids`

Reply:
Ok i misunderstood the utility of this line, my bad, i've added it back